desktop/apps/the_armory_show/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import queryString from 'query-string'
+import merge from 'lodash.merge'
+import { renderLayout } from '@artsy/stitch'
+
+import adminOnly from '../../lib/admin_only'
+import JSONPage from '../../components/json_page/es6'
+import ArmoryShowFairWeekPage from './components/ArmoryShowFairWeekPage'
+
+const SLUG = 'the-armory-show-fair-week'
+
+class EditableArmoryShowFairWeekPage extends JSONPage {
+  registerRoutes() {
+    this.app.get(this.jsonPage.paths.show, adminOnly, this.show.bind(this))
```

Comment:
We put an adminOnly filter on this so that no one externally can access this page yet.